pom.xml

```diff
@@ -111,6 +111,16 @@
               </plugin>
 		  </plugins>
 		</pluginManagement>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
+				</configuration>
+			</plugin>
+		</plugins>
```

Comment:
Unless this has changed very recently, Jenkins plugins are supposed to be compiled for 1.7 compatibility.